01/index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    ¿Qué es git?
+    Es un manejador o controlador de versiones.
+
+    ¿Para qué sirve?
+    Para poder hacer cambios sin conexión de forma remota fácilmente, además sirve para comparar los cambios que se han hecho con versiones anteriores.
```

Comment:
if git is a version controller what's the purpose of git? Do we need some internet connection?